js/admin.js

```diff
@@ -0,0 +1,13 @@
+jQuery( document ).ready( function ( $ ) {
```

Comment:
I don't love a dependence on jQuery. is jQuery event binding the best way to handle this? Would a callback function on an onclick be more traditional? I admit I am not up to speed on how core prefers doing this these days.